packages/graphiql-explorer/src/lib/utils.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable */
+// cSpell:disable
+
+import {
+  isWrappingType,
+  isNonNullType,
+  isInputObjectType,
+  isRequiredInputField,
+  Kind,
+  isLeafType,
+  isEnumType,
+  parse,
+} from 'graphql';
+
+import type {
+  ObjectFieldNode,
+  GraphQLScalarType,
+  GraphQLEnumType,
+  ValueNode,
+  GraphQLArgument,
+  GraphQLInputField,
+  GraphQLOutputType,
+  GraphQLInputType,
+  OperationDefinitionNode,
+  DocumentNode,
+  GraphQLObjectType,
+} from 'graphql';
+
+export function unwrapOutputType(outputType: GraphQLOutputType) {
+  let unwrappedType = outputType;
+  while (isWrappingType(unwrappedType)) {
+    unwrappedType = unwrappedType.ofType;
+  }
+  return unwrappedType;
+}
+
+export function unwrapInputType(inputType: GraphQLInputType) {
+  let unwrappedType = inputType;
+  while (isWrappingType(unwrappedType)) {
+    unwrappedType = unwrappedType.ofType;
+  }
+  return unwrappedType;
+}
+export function isRequiredArgument(arg: GraphQLArgument): boolean {
+  return isNonNullType(arg.type) && arg.defaultValue === undefined;
+}
+
+export function capitalize(string: string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function parseQuery(text: string): null | DocumentNode | Error {
+  try {
+    if (!text.trim()) {
+      return null;
+    }
+    return parse(
+      text,
+      // Tell graphql to not bother track locations when parsing, we don't need
```

Comment:
```suggestion
      // Tell graphql to not bother tracking locations when parsing; we don't need
```

Reply:
thanks, Ted!

I'm trying to change as little as possible from the original project in this PR, but this change makes sense of course